### PR TITLE
Creation of page for PyHEP activities and info

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
General page under "working groups" for all PyHEP activities:

- Not just the workshop in Sofia.
- Gitter channel.
- GitHub repo on resources.

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
One could also think about moving the advert and details on PyHEP 2018 to a separate file, say under a directory `pyhep/2018/`, as done similarly for GSoC. Or to `pyhep/pyhep201[8-...].html` files under a common directory. Dunno, just thinking out loud ...

File: _workinggroups/pyhep.md [hunk 1]
```diff
@@ -1,14 +1,36 @@
...
+the sharing of knowledge and expertise. It embraces the broad community, from HEP to the Astroparticle and Intensity Frontier communities.
+
+The group is currently coordinated by Chris Tunnell (Rice U.), Eduardo Rodrigues (U. Cincinnati) and Graeme Stewart (CERN).
+All coordinators can be reached via <hsf-pyhep-organisation@googlegroups.com>.
+
+# Getting Involved
+
+Everyone is welcome to join the community and participate by means of the following:
+
+[Gitter channel PyHEP](https://gitter.im/HSF/PyHEP){:target="_pyhep_gitter_channel} for any informal exchanges.
```
Comment (graeme-a-stewart) [inline _workinggroups/pyhep.md]: Wouldn't it be better to make this all a bulleted list? The formatting looks quite strange having this as just a pagaraph, following a colon above.

File: _workinggroups/pyhep.md [hunk 2]
```diff
@@ -1,14 +1,36 @@
...
+
+[GitHub repository of resources](https://github.com/hsf-training/PyHEP-resources){:target="_pyhep_resources_repo"}
```
Comment (graeme-a-stewart) [inline _workinggroups/pyhep.md]: Bullet

File: _workinggroups/pyhep.md [hunk 3]
```diff
@@ -1,14 +1,36 @@
...
+- Python libraries of interest to Particle Physics.
```
Comment (graeme-a-stewart) [inline _workinggroups/pyhep.md]: I'm not sure why this text exists - it doesn't add much to what's on the line above

**Comment by graeme-a-stewart** [comment]
Thanks @eduardo-rodrigues - looks very good, barring a few small points about bulleting what looks like to me is a list.

I am not worried about having the workshop material here at the moment, but I would put it into the past tense. Once we have a second workshop we can think about a separate page.

File: _workinggroups/pyhep.md (hunk 1 above)
Comment (eduardo-rodrigues) [reply _workinggroups/pyhep.md]: Sure, will do.

File: _workinggroups/pyhep.md (hunk 3 above)
Comment (eduardo-rodrigues) [reply _workinggroups/pyhep.md]: It's just extra info on the bullet point, not an extra line.

File: _workinggroups/pyhep.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+1. Historical perspective and overview
+2. PyROOT and Python bindings
+3. HEP python software ecosystem
+4. Distribution and installation
+5. Analysis frameworks
+6. Python 2 and 3
+7. Open discussion on education and training
+
+There was no training sessions nor hackathons in this first workshop,
```
Comment (graeme-a-stewart) [inline _workinggroups/pyhep.md]: *were* no training sessions

Comment (eduardo-rodrigues) [reply _workinggroups/pyhep.md]: Oops, sorry. Correcting ….

**Pull request merged**